### PR TITLE
drivers: wifi: Fix for occasional RPU lockup

### PR DESCRIPTION
**Pull request by muraliThokala**
SHEL-1357: Fix the occasional RPU lockup when Wi-Fi and SR coexistence is enabled.

### CODE REVIEW

File: drivers/wifi/nrf700x/zephyr/src/zephyr_coex.c [hunk 1]
```diff
@@ -47,15 +47,15 @@ static const struct gpio_dt_spec btrf_switch_spec =
...
-	0x19, 0xF6, 0x8, 0x62, 0xF5,
-	0xF5, 0x19, 0x19, 0x74, 0x74,
-	0x8, 0x1E2, 0xD5, 0xD5, 0x1F6,
-	0x1F6, 0x61, 0x61, 0x1E2, 0x8,
-	0x1F6, 0x1F6, 0x19, 0x19, 0x1E2,
-	0x8, 0xF5, 0xF5, 0xD5, 0xD5,
-	0x8, 0x1E2, 0x51, 0x51, 0x74,
-	0x74, 0xF6, 0x19, 0x62, 0x19,
-	0xF6, 0x8, 0x62, 0x8, 0x1A
+	0x0019, 0x00F6, 0x0008, 0x0062, 0x00F5,
```
Comment (krish2718) [inline drivers/wifi/nrf700x/zephyr/src/zephyr_coex.c:50]: can you please do this change in a separate commit?

Comment (muraliThokala) [reply drivers/wifi/nrf700x/zephyr/src/zephyr_coex.c:50]: Done.

**Review by krish2718 — APPROVED** [review]

**Review by sachinthegreen — APPROVED** [review]
